hugegraph-core/src/main/java/org/apache/hugegraph/traversal/algorithm/KoutTraverser.java

```diff
@@ -138,6 +138,38 @@ public KoutRecords customizedKout(Id source, EdgeStep step,
         return records;
     }
 
+    public KoutRecords DFSKout(Id source, Steps steps,
+                               int maxDepth, boolean nearest,
+                               long capacity, long limit) {
+        E.checkNotNull(source, "source vertex id");
+        this.checkVertexExist(source, "source vertex");
+        checkPositive(maxDepth, "k-out max_depth");
```

Comment:
"this." -- keep the same styles

Reply:
The difference between these two `check` method is that `checkPositive()` is a static method while `checkVertexExist()` is not.

According to stackoverflow, `this` refers to the object instance and there is no object instance in a call of a static method.

stackoverflow: https://stackoverflow.com/questions/11664522/why-cant-we-use-this-keyword-in-a-static-method

Reply:
I am not sure, so should we add `this.` keyword? 

I checked other traverser implementations and they didn't add `this.` keyword, such as PathsTraverser.java, SameNeighborTraverser.java and so on.

Reply:
> I am not sure, so should we add `this.` keyword?
> 
> I checked other traverser implementations and they didn't add `this.` keyword, such as PathsTraverser.java, SameNeighborTraverser.java and so on.

seems if `checkPositive` is a static method, we shouldn't add `this` for it

Reply:
It's OK, I missed your reply